solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/PerReplicaStatesFetcher.java

```diff
@@ -35,6 +36,8 @@ public class PerReplicaStatesFetcher {
   public static PerReplicaStates fetch(
       String path, SolrZkClient zkClient, PerReplicaStates current) {
     try {
+      assert CommonTestInjection.injectBreakpoint(
```

Comment:
Why we need to assert here?

Reply:
so that for normal execution (non unit test), this call will be ignored